extension/lib/doc.js

```diff
@@ -10,7 +10,10 @@ if (window[namespace] === true) {
 var settings = window.settings;
 
 // If the number of sentences in the doc exceeds this, mark the document as unreadable.
-const MAX_NUM_SENTENCE = 2000;
+const MAX_NUM_SENTENCE = 4000;
+// Any word with a TF-IDF score above this frequency is a keyword
+// 0.001-0.005 give reasonable values; high filter, less keywords
+const keywordFilter = 0.001; 
```

Comment:
Current keyword filter, will post pics soon showing what this is like. Higher filter, less keywords. Based on TF-IDF score